subprojects/tooling-api/src/testFixtures/groovy/org/gradle/integtests/tooling/fixture/ProgressEvents.groovy

```diff
@@ -75,7 +75,7 @@ class ProgressEvents implements ProgressListener {
 
                     // Display name should be mostly unique
                     if (!skipValidation && uniqueBuildOperation(descriptor)) {
-                        if (descriptor.displayName in ['Configure settings', 'Configure build', 'Calculate task graph', 'Run tasks']) {
+                        if (descriptor.displayName in ['Run build', 'Configure settings', 'Configure build', 'Calculate task graph', 'Run tasks']) {
```

Comment:
'Run build' now appears twice in a regular build, just like the others on this line. Once for the whole build, once for `buildSrc`.